src/interfaces/store-interfaces/state-interfaces/authorization-proof-interface.ts

```diff
@@ -4,4 +4,5 @@ export interface AuthorizationProofIF {
     fileKey: string
     fileName: string
   }>
+  consumerDocumentId?: string
```

Comment:
Can we make this name consistent with `documentServiceId` (in filing-interfaces.ts)?

Reply:
We have both documentServiceId and consumerDocumentId as well.
Thanks

Reply:
I was thinking about this documentServiceId + documentConsumerId.

Is the first one "service id" and second one "document id"? If yes then what you have makes sense.